sej-videos.html

```diff
@@ -100,4 +123,15 @@ <h1>Inspirational Videos</h1>
  .wellbeing {
     width: 100%;
  }
+ 
+ #poster{
```

Comment:
Instead of manually setting widths, we can use bootstrap grids that are already available to us. 
For example https://github.com/Muhammad-Ali-Ahmad/mka-interactive/blob/master/wellbeing.html#L7 here we are using col-sm-4 class which is same as 1/3 width. The way they (bootstrap) have made grids, they have 12 columns so if we do col-sm-4 then it means we can have 3 items (4x3=12) in a row. If we make that col-sm-3 then we can have 4 items (e.g. education page).
In this case we can use col-sm-4 like on wellbeing page or even col-sm-6. I hope this makes sense :)